docs/user/rstudio/ide/guide/publish/connecting.qmd

```diff
@@ -2,34 +2,46 @@
 title: "Connecting"
 ---
 
-Adapted from <https://docs.rstudio.com/connect/user/connecting/>
+::: {.callout-warning}
+This section is not complete until we have updated screenshots for publishing to Posit Connect.
+:::
+
+RStudio provides "Push button deployment" to Posit Connect, Shinyapps.io, and RPubs.com. [Posit Connect](https://www.posit.co/products/connect/) supports a spectrum of data products, static or dynamic, developed in R and Python: Dashboards, applications, APIs, reports, and so much more. Connect enables flexible permissions configurations and meets all enterprise security requirements. Shinyapps.io supports hosting of Shiny for R and Python, and RPubs.com supports hosting of static-content only.
```

Comment:
By the next (currently in progress) release, we will also have Push button deployment to rstudio.cloud (to be renamed posit.cloud, I believe) so you may want to add that in here

Reply:
You might want to check with the Hosted team, but posit.cloud should support Shiny for R and Python, Plumber APIs, and RMarkdown content

Reply:
You may want/need to redo these screenshots to reflect the rstudio.cloud/posit.cloud, however I think it might also be a beta, so again I would ask the Hosted team whether we want to include these in documentation

Reply:
I'm waiting on information from them - thanks for the prompt!